vyper/ast/pre_parser.py

```diff
@@ -114,13 +114,22 @@ def pre_parse(code: str) -> Tuple[ClassTypes, str]:
 
             if typ == NAME and string == "class" and start[1] == 0:
                 raise SyntaxException(
-                    "The `class` keyword is not allowed. Perhaps you meant `contract` or `struct`?",
+                    "The `class` keyword is not allowed. "
+                    "Perhaps you meant `interface` or `struct`?",
                     code,
                     start[0],
                     start[1],
                 )
 
-            # Make note of contract or struct name along with the type keyword
+            if typ == NAME and string == "contract" and start[1] == 0:
+                raise SyntaxException(
+                    "The `contract` keyword has been deprecated. Please use `interface`?",
```

Comment:
Is that question mark intentional?

Reply:
No lol

Reply:
I think I was trying to say "did you mean to use interface?"